<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TotalReport</class>
 <widget class="QWidget" name="TotalReport">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>45</number>
     </property>
     <property name="rightMargin">
      <number>45</number>
     </property>
     <item>
      <widget class="QPushButton" name="BackButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>НАЗАД</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>300</number>
       </property>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="EcoTaxi">
         <property name="text">
          <string>ОТЧЁТ</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="Header">
         <property name="text">
          <string>ОБЩИЙ</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>410</number>
     </property>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>от </string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="FromDateButton">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>01.01.0001</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>до </string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="ToDateButton">
       <property name="minimumSize">
        <size>
         <width>220</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>02.02.0002</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="FilterButton">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>Фильтр</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>45</number>
     </property>
     <property name="rightMargin">
      <number>800</number>
     </property>
     <item>
      <widget class="QTableView" name="tableView">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
     <property name="spacing">
      <number>60</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>45</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="detailButton">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>52</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>52</height>
        </size>
       </property>
       <property name="text">
        <string>Детали</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="ToEXCELButton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string>ЭКСПОРТ В EXCEL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ToPDFButton">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>52</height>
          </size>
         </property>
         <property name="text">
          <string>ЭКСПОРТ  В PDF</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
